.github/workflows/ci.yml

```diff
@@ -401,6 +401,14 @@ jobs:
       - name: build
         run: yarn run build
 
+      - name: build:types
+        # Concurrency is 1 to avoid bizarre cross-package contamination.
+        run: yarn lerna run generate:types
+
       - name: pack
         # Concurrency is 1 to avoid bizarre cross-package contamination.
-        run: yarn run lerna exec --concurrency=1 yarn pack
+        run: yarn lerna run pack
+
+      - name: clean:types
+        # Concurrency is 1 to avoid bizarre cross-package contamination.
+        run: yarn lerna run clean:types
```

Comment:
Oh, and `yarn lerna run pack` does absolutely nothing if there is no explicit `packageJson.scripts.pack` lifecycle script:
```
$ yarn lerna run pack
yarn run v1.22.19
$ /Users/michael/agoric/endo/node_modules/.bin/lerna run pack
lerna notice cli v5.6.2
lerna info versioning independent
lerna success run No packages found with the lifecycle script 'pack'
✨  Done in 0.76s.
$
```
Please revert to `yarn lerna exec yarn pack` or simply just `yarn pack`.

Reply:
Ahah! Thanks for the tip. I’m going to continue using `yarn lerna run` for `build:types` and `clean:types` since ignoring the step on packages that provide no such script is a useful behavior, and I’m reverting to your original lines for `pack`.

Reply:
Also, I don’t understand the concurrency concern. It’s definitely the case that `build:types` needs to run from the bottom up, but it could do so concurrently. With `build:types` occurring in a separate step, we could probably safely pack concurrently as well. To that end, I’m going to remove the concurrency limit and comment; please let me know if they remain necessary.